src/options/ProxyList.js

```diff
@@ -28,27 +28,30 @@ export class ProxyList {
 
   view () {
     const items = this.model.list.map(this.renderProxyItem.bind(this))
-    const actions = m('div', [
-      m('a[href=/proxies/new]', { oncreate: m.route.link, class: style.primaryButton }, '+')
+    const actions = m('div.proxy-list-item', [
+      m('.proxy-name'),
```

Comment:
seems like left here by accident.

Reply:
As I understood it is to have the same look. 
I did some CSS copypaste, but made it have a distinct class.